tool/teleport/common/teleport.go

```diff
@@ -444,6 +446,22 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	joinOpenSSH.Flag("debug", "Enable verbose logging to stderr.").Short('d').BoolVar(&ccf.Debug)
 
 	integrationCmd := app.Command("integration", "Integration commands")
+	integrationBootstrapCmd := integrationCmd.Command("bootstrap", "Bootstrap an integration")
+	integrationBootstrapCreateExternalCloudAuditCmd := integrationBootstrapCmd.Command("externalcloudaudit", "Bootstraps external cloud audit infrastructure.")
+	integrationBootstrapCreateExternalCloudAuditCmd.Flag("aws-region", "The region to use. Overrides config/env settings").StringVar(&ccf.
+		IntegrationBootstrapCreateExternalCloudAuditArguments.Region)
+	integrationBootstrapCreateExternalCloudAuditCmd.Flag("long-term-storage-bucket", "S3 Bucket for long term storage of audit events and session recordings.").
+		Required().StringVar(&ccf.IntegrationBootstrapCreateExternalCloudAuditArguments.LongTermStorageBucket)
+	integrationBootstrapCreateExternalCloudAuditCmd.Flag("transient-bucket", "S3 bucket for transient storage of athena query results and large payloads.").
+		Required().StringVar(&ccf.IntegrationBootstrapCreateExternalCloudAuditArguments.TransientBucket)
+
+	integrationBootstrapCreateExternalCloudAuditCmd.Flag("athena-workgroup", "Name of athena workgroup").Default("teleport").StringVar(&ccf.
+		IntegrationBootstrapCreateExternalCloudAuditArguments.AthenaWorkgroup)
+	integrationBootstrapCreateExternalCloudAuditCmd.Flag("glue-database-name", "Name of the glue database to create").Default("teleport").StringVar(&ccf.
+		IntegrationBootstrapCreateExternalCloudAuditArguments.DatabaseName)
+	integrationBootstrapCreateExternalCloudAuditCmd.Flag("glue-table-name", "Name of the glue table to create").Default("auditevents").StringVar(&ccf.
+		IntegrationBootstrapCreateExternalCloudAuditArguments.TableName)
```

Comment:
I met with the design team yesterday and we agreed it would be preferable to have a single one-off command to handle both the resource bootstrap and creating the IAM permissions. Instead of a whole bunch of arguments I think I'd like to pass the entire `ExternalCloudAudit` config as a JSON blob. Maybe leave the CLI command out of this PR, I will handle combining it with the IAM permissions command I already created, if you can just write the function that handles creating all the resources, taking [this](https://github.com/gravitational/teleport/blob/2087a2fda8d57fe1d01e27b26da331daf5de8b94/api/types/externalcloudaudit/externalcloudaudit.go#L32) `ExternalCloudAudit` type and the region as input. Eventually I think I will actually add the region to that type, we should store it somewhere.